Buildings/Experimental/OpenBuildingControl/ASHRAE/G36/Atomic/EconDamperPositionLimitsMultiZone.mo

```diff
@@ -36,23 +36,23 @@ block EconDamperPositionLimitsMultiZone
     annotation (Placement(transformation(extent={{-220,-120},{-180,-80}}),
         iconTransformation(extent={{-120,-10},{-100,10}})));
 
-  CDL.Interfaces.RealOutput yOutDamPosMin(min=0, max=1, unit="1")
+  CDL.Interfaces.RealOutput yOutDamPosMin(final min=outDamPhyPosMin, final max=outDamPhyPosMax, final unit="1")
     "Minimum outdoor air damper position limit"
```

Comment:
@milicag it would be more readable to make each assignment occupied each line:
`final min=outDamPhyPosMin, `
`final max=outDamPhyPosMax, `
 `final unit="1"`